src/css/components/Feedback.scss

```diff
@@ -0,0 +1,120 @@
+@import "../config/index.scss";
```

Comment:
Do we need to do this still? I think we were doing this for the sake of CSS modules in LFE.

Reply:
yeah we could probably drop it - just keeping this file consistent with the rest